### PR TITLE
chore(ci): update labelbot docs glob

### PR DESCRIPTION
**Pull request by leotm**
Since root repo docs only living in root dir

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -88,7 +88,7 @@
   - changed-files:
       - any-glob-to-any-file:
           - '**/*.md'
-          - '**/docs/**'
+          - 'docs/**'
```
Comment (leotm) [inline .github/labeler.yml:91]: or remove it and let all .md's be picked up from

https://github.com/LavaMoat/LavaMoat/blob/716ced82813c94afdd255d7db443475959488203/.github/labeler.yml#L90

Comment (boneskull) [reply .github/labeler.yml:91]: The intent here was to pick up _any_ markdown file (except snapshots) and any present or future docs.

Comment (leotm) [reply .github/labeler.yml:91]: agreed ^ i think `docs/**` still makes sense for our repo (like in the [examples](https://github.com/actions/labeler/blob/main/README.md))

Screenshot 2024-01-18 at 3 51 12 pm

but if we plan on recursive docs folders in future (e.g. `packages/aa/docs/newDoc.md`) then i'll go ahead and close this pr

Comment (boneskull) [reply .github/labeler.yml:91]: yeah, it's just so we don't need to change it later, I guess.  tbh I don't see any harm in leaving it as-is

**Pull request closed without merging**